doc/en/pom.xml

```diff
@@ -24,234 +24,426 @@
-            <goals>
-              <goal>generate</goal>
-            </goals>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/system-status.yaml</inputSpec>
+              <output>${project.build.directory}/api/system-status</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>templates</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
             <configuration>
               <inputSpec>${project.basedir}/api/1.0.0/templates.yaml</inputSpec>
-              <language>html</language>
               <output>${project.build.directory}/api/templates</output>
             </configuration>
           </execution>
+          <execution>
+            <id>transforms</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/transforms.yaml</inputSpec>
+              <output>${project.build.directory}/api/transforms</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>usergroup</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/usergroup.yaml</inputSpec>
+              <output>${project.build.directory}/api/usergroup</output>
+            </configuration>
+          </execution>
           <execution>
             <id>wmslayers</id>
             <phase>process-resources</phase>
-            <goals>
-              <goal>generate</goal>
-            </goals>
+            <goals><goal>generate</goal></goals>
             <configuration>
               <inputSpec>${project.basedir}/api/1.0.0/wmslayers.yaml</inputSpec>
-              <language>html</language>
               <output>${project.build.directory}/api/wmslayers</output>
             </configuration>
           </execution>
           <execution>
             <id>wmsstores</id>
             <phase>process-resources</phase>
-            <goals>
-              <goal>generate</goal>
-            </goals>
+            <goals><goal>generate</goal></goals>
             <configuration>
               <inputSpec>${project.basedir}/api/1.0.0/wmsstores.yaml</inputSpec>
-              <language>html</language>
               <output>${project.build.directory}/api/wmsstores</output>
             </configuration>
           </execution>
           <execution>
             <id>wmtslayers</id>
             <phase>process-resources</phase>
-            <goals>
-              <goal>generate</goal>
-            </goals>
+            <goals><goal>generate</goal></goals>
             <configuration>
               <inputSpec>${project.basedir}/api/1.0.0/wmtslayers.yaml</inputSpec>
               <language>html</language>
```

Comment:
Do we still want the HTML here?

Reply:
No I moved it up as common configuration.